Model/Indexer/DataProvider/AsyncEventSubscriberLogs.php

```diff
@@ -25,14 +20,13 @@ public function __construct(
     /**
      * Get async event logs by log ids
      *
-     * @param array $logIds
      * @param string $asyncEvent
+     * @param array|null $logIds
      * @return Collection
      */
-    public function getAsyncEventLogs(array $logIds, string $asyncEvent): Collection
+    public function getAsyncEventLogs(string $asyncEvent, array $logIds = null): Collection
```

Comment:
Should `$logIds = []` if unset, instead of `null`? Or type the argument with `?array`